scripts/create_mask.py

```diff
@@ -346,4 +347,4 @@ def parse_input_arguments(argv):
 
 
 if __name__ == '__main__':
-    main(sys.argv)
```

Comment:
I prefer you undo this particular change. If `main(argv)` then you can write unit tests against `main()`

Reply:
I'm not sure how to invoke that via scripts in the pyproject.toml.

I suppose I can make a wrapper method w/o/ args that calls the other method with args?

Reply:
For then entrypoint you could change the signature to
```py
def main(args=None):
   if args is None:
     args